website/docs/r/container_group.html.markdown

```diff
@@ -80,24 +80,28 @@ The following arguments are supported:
 
 * `dns_name_label` - (Optional) The DNS label/name for the container groups IP. Changing this forces a new resource to be created.
 
+* `dns_name_label_reuse_policy` - (Optional) The value representing the security enum. `Noreuse`, `ResourceGroupReuse`, `SubscriptionReuse`, `TenantReuse` or `Unsecure`. Defaults to `Unsecure`. Changing this forces a new resource to be created.
+
 ~> **Note:** DNS label/name is not supported when deploying to virtual networks.
 
 * `exposed_port` - (Optional) Zero or more `exposed_port` blocks as defined below. Changing this forces a new resource to be created. 
 
 ~> **Note:** The `exposed_port` can only contain ports that are also exposed on one or more containers in the group. 
 
-* `ip_address_type` - (Optional) Specifies the IP address type of the container. `Public`, `Private` or `None`. Changing this forces a new resource to be created. If set to `Private`, `network_profile_id` also needs to be set.
+* `ip_address_type` - (Optional) Specifies the IP address type of the container. `Public`, `Private` or `None`. Changing this forces a new resource to be created. If set to `Private`, `subnets` also needs to be set.
 
 ~> **Note:** `dns_name_label` and `os_type` set to `windows` are not compatible with `Private` `ip_address_type`
 
 * `key_vault_key_id` - (Optional) The Key Vault key URI for CMK encryption. Changing this forces a new resource to be created.
 
-* `network_profile_id` - (Optional) Network profile ID for deploying to virtual network.
+* `subnets` - (Optional) The subnet resource IDs for a container group.
```

Comment:
Can we also add: `Changing this forces a new resource to be created.` Only one element is supported at this moment?